components/upload/events.md

```diff
@@ -28,7 +28,7 @@ This article explains the events available in the Telerik Upload for Blazor:
 
 The `OnSelect` event fires every time the user selects new files for upload. The event arguments provide the list of newly selected files.
 
-You can cancel the event based on a condition (for example, some information about the selected files) so that those files will not be populated in the Upload component.
+You can cancel the event based on a condition (e.g. some information about the selected files), so that those files will not be populated in the Upload component.
```

Comment:
```suggestion
You can cancel the event based on a condition (for example, some information about the selected files), so that those files will not be populated in the Upload component.
```